.gitignore

```diff
@@ -1,5 +1,5 @@
 # IDE
-.vscode
+#.vscode
```

Comment:
It is a fairly common practice these days to include the .vscode directory, and a couple of the core files (recommended extensions, launch/debug settings, taskbar tasks, workspace settings to hide generated files, etc.). 

Let me know if you have concerns with this though

Reply:
No concerns here, I just filtered it out because I know that some subset of people still haven't discovered how awesome VSCode is and I didn't want to add items that could be perceived as clutter in the code base.